.storybook/theme.scss

```diff
@@ -1 +1,102 @@
-@use '../scss.scss';
+@use '../scss.scss' with (
+	$global: (
+		'blue-0': #EBEFFF,
+		'blue-1': #B0C3F2,
+		'blue-2': #7AA2E5,
+		'blue-3': #4A89D8,
+		'blue-4': #1E78CB,
+		'blue-5': #1458A9,
+		'blue-6': #0C3D87,
+		'blue-7': #062665,
+		'blue-8': #021443,
+		'blue-9': #000721,
+
+		'red-0': #FF8A8A,
+		'red-1': #E65555,
+		'red-2': #CE2929,
+		'red-3': #B50505,
+		'red-4': #9C0000,
+		'red-5': #840000,
+		'red-6': #6B0000,
+		'red-7': #520000,
+		'red-8': #3A0000,
+		'red-9': #210000,
+
+		'orange-0': #FA714B,
+		'orange-1': #E24F23,
+		'orange-2': #C93302,
+		'orange-3': #B12E00,
+		'orange-4': #982A00,
+		'orange-5': #802500,
+		'orange-6': #682000,
+		'orange-7': #4F1900,
+		'orange-8': #371200,
+		'orange-9': #1F0B00,
+
+		'green-0': #80D971,
+		'green-1': #57C244,
+		'green-2': #34AB1F,
+		'green-3': #189402,
+		'green-4': #137D00,
+		'green-5': #0F6600,
+		'green-6': #0C4F00,
+		'green-7': #083800,
+		'green-8': #052100,
+		'green-9': #020A00,
+
+		'neutral-0': #E6EFFF,
```

Comment:
What does `neutral` mean? Is it a color?

Reply:
This is another area that is a bit gray 😂  Neutrals are grays and sometimes black/white grouped together. They are usually separated in this way because neutrals tend to power most of the interface design and typography. I don't feel strongly about the name neutral. In ADS we categorize our `$gray` tokens under the headline `Neutrals` https://design.arcxp.com/patterns/color